.github/workflows/bot-bot.yml

```diff
@@ -81,7 +80,7 @@ jobs:
           export RUN_URL="https://github.com/regro/cf-scripts/actions/runs/${RUN_ID}"
           conda-forge-tick deploy-to-github
         env:
-          BOT_TOKEN: ${{ secrets.AUTOTICK_BOT_TOKEN }}
```

Comment:
Explanation: This only pushes to cf-graph so the permissions for regro are sufficient